### ISSUE TITLE
[Docs]: Update Docs for `grafana_machine_learning_alert`

### ISSUE THREAD

**Issue by SeamusGrafana** [grafana/terraform-provider-grafana#1709]
Update the Docs for `grafana_machine_learning_alert` to clarify usage.

- Specify that one of `job_id` or `outlier_id` is required
- Specify that `anomaly_condition` can only be used with `job_id`
- Specify the acceptable values for `anomaly_condition` one of `any`, `high`, `low` and what each does.
- Specify the acceptable values for `no_data_state` one of `Alerting`, `NoData` or `OK`
- Correct Import Example to show Format: `Import ID must be in the format '/(jobs|outliers)/<jobID>/alerts/<alertID>'`
- Correct `outlier_id (String) The forecast this alert belongs to.` to `outlier_id (String) The outlier this alert belongs to.`

**Comment by SeamusGrafana** [comment]
Cc: @csmarchbanks

**Comment by csmarchbanks** [comment]
I'm honestly disappointed that terraform doc generation doesn't handle this for us. All of those are already specified in the code and writing extra docs seems tedious and bound to get out of sync. Regardless, feel free to open a PR, or else I'll get to it soon.

**Comment by julienduchesne** [comment]
> I'm honestly disappointed that terraform doc generation doesn't handle this for us. All of those are already specified in the code and writing extra docs seems tedious and bound to get out of sync. Regardless, feel free to open a PR, or else I'll get to it soon.

Yeah, they definitely could've made this better with the new plugin framework. We have a helper for some of these: [AllowedValuesDescription](https://github.com/grafana/terraform-provider-grafana/blob/main/internal/common/schema.go#L56)

**Comment by YerrabothulaNavyaSree09** [comment]
Hi, I would like to work on this issue.
Can you please confirm if it’s still available? I’d love to help update the documentation for grafana_machine_learning_alert.
